crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -27,6 +27,19 @@ fn convert_level(level: LogLevel) -> Level {
     }
 }
 
+/// Namespace for pure functions provided by the SDK.
+pub(crate) mod bitwarden_pure {
+    use super::*;
+
+    pub(crate) fn setup_once() {
+        console_error_panic_hook::set_once();
+        let log_level = convert_level(LogLevel::Info);
+        if let Err(_e) = console_log::init_with_level(log_level) {
+            set_max_level(log_level.to_level_filter())
+        }
+    }
```

Comment:
issue: Can we move/remove this? We want to eventually add more control over the initialization process and this will make it much harder to coordinate that. I don't think calling `symmetric_decrypt()` should implicitly mean setting the logging level to `Info`. For context I don't think setting it in `BitwardenClient::new` is the right way either. I like what you did with the `SdkLoadService` and I think we should try to move the `init` there instead. I also think that consumers of the SDK in the clients might need to depend on the SDK initialization somehow and not just assume that WASM has been initialized by the time their function runs

Reply:
This is kind of tricky.

The issue is that these methods are otherwise pure and don't need to depend on any previous calls or state. In the case of the panic hook, I think what is here is fine. The documentation specifically calls out subsequent invocations doing nothing.

All that can be done with your suggesting is taking an optional `LogLevel` in the signature of the pure functions and passing it along here. I didn't think that was worthwhile.